src/sound/musicplayer.js

```diff
@@ -2,7 +2,7 @@ import * as $j from 'jquery';
 
 export class MusicPlayer {
 	constructor() {
-		this.audio = $j('#audio')[0];
+		this.audio = $j('#audio-player')[0];
```

Comment:
There was a duplicate ID clash with the UI button for opening the music player screen.